src/ray/core_worker/experimental_mutable_object_manager.cc

```diff
@@ -340,6 +372,7 @@ Status MutableObjectManager::ReadRelease(const ObjectID &object_id)
 }
 
 Status MutableObjectManager::SetError(const ObjectID &object_id) {
+  RAY_LOG(DEBUG) << "Set error " << object_id;
```

Comment:
Nit: reader might think something went wrong, how about "SetError on " << object_id

Reply:
Ah will change, but FYI this is DEBUG level so users shouldn't see it.